apps/client/src/components/Visual.js

```diff
@@ -23,7 +23,7 @@ export default ({ title, src, ...rest }) => {
         onLoad={() => setLoaded(true)}
         onError={handleError}
       />
-      {title && !errored && (
+      {title && (
```

Comment:
Maybe we could also handle this caption as alt in the same PR

Reply:
Let's do this in another PR